installation/linux.md

```diff
@@ -48,17 +48,17 @@ Alternatively resort to the [manual installation approach](linux.html#manual-ins
 Choose between the latest Beta release or a Snapshot with all incoming contributions, rebuild on [every change](https://oss.jfrog.org/webapp/#/builds/openHAB-Distribution).
 As openHAB 2 is still in an evolving state, the snapshot may be the **preferred choice**.
 
-The repository URLs will be stored in the file `/etc/apt/sources.list.d/openhab.list`.
+The repository URLs will be stored in the file `/etc/apt/sources.list.d/openhab2.list`.
 Be careful to not have conflicting repositories in your sources list.
```

Comment:
> I had this thought when writing the docs too.

Great minds think alike. :-)

I plan on having a "clean-up" step in the migration docs that covers removal of OH 1 and it will 
include removal of the old openhab.list files among other things.

I'm trying to structure the migration steps so the OH 1 install is not broken until the OH 2 install is completed and proven to work.

If you know of other files or folders that should be removed or renamed let me know so I don't miss them in the setup. With your work on openHABian you might be one of the most knowledgeable on that front.